src/modules/enrollment/components/EnrollmentAssessmentActionButtons.tsx

```diff
@@ -41,6 +45,17 @@ const NewAssessmentMenu: React.FC<Props> = ({ enrollment }) => {
   const enrollmentId = enrollment.id;
   const clientId = enrollment.client.id;
 
+  const [getEligability, { data, error, loading }] =
+    useGetClientAssessmentEligibilitiesLazyQuery({
+      fetchPolicy: 'cache-and-network',
+    });
+
+  useEffect(() => {
+    getEligability({ variables: { clientId, enrollmentId } });
+  }, [enrollment.client.lockVersion, clientId, enrollmentId, getEligability]);
```

Comment:
oh, why did you add the lock version like this?

Reply:
This was a quick way to ensure the query is run again after an assessment is added. I'll revisit this and see if it's still necessary

Reply:
looks like it's not needed with a cache-and-network fetch policy